week-8/Homework/mandatory/4-programmer-humour/script.js

```diff
@@ -0,0 +1,17 @@
+console.log(document.querySelector("body"));
+
+async function getComix() {
+  const response = await fetch("https://xkcd.now.sh/?comic=latest");
+  const data = await response.json();
+  console.log(data);
+  let divContainer = document.getElementById("imageContainer");
+  let imgItem = document.getElementById("comicImage");
+  imgItem.src = data.img;
+}
+
+window.onload = getComix().catch((error) => {
```

Comment:
I think the `window.onload` is not doing much here, because you are calling the `getComix` function immediately, rather than passing it as a reference.

So the function gets called as soon as the code is run, not in response to the onload event.

Reply:
I like the combined use of async/await with .catch() for error handling, didn't know you could do that!